cdap-docs/reference-manual/source/http-restful-api/artifact.rst

```diff
@@ -0,0 +1,520 @@
+     - Version of the artifact
+   * - ``<plugin-type>``
+     - Type of the plugin
+   * - ``<plugin-name>``
+     - Name of the plugin
+   * - ``<scope>``
+     - Optional scope filter. If not specified, defaults to 'user'.
+
+This will return a JSON array that lists the plugins of the specified type and name
+available to the artifact. Each element in the array is a JSON object containing
+the artifact that the plugin originated from, and the plugin's class name, description, name, type, and properties.
+Example output for the ``ScriptFilter`` plugin available to version |release|
+of the ``cdap-etl-batch`` artifact (pretty-printed and reformatted to fit):
+
+.. container:: highlight
+
+  .. parsed-literal::
+    |$| GET <base-url>/namespaces/default/artifacts/cdap-etl-batch/versions/|release|/extensions/transform/plugins/ScriptFilter?scope=system
+
+    [
+      {
+        "artifact": {
+            "name": "cdap-etl-lib",
+            "scope": "SYSTEM",
+            "version": "|release|-batch"
+        },
+        "className": "co.cask.cdap.etl.transform.ScriptFilterTransform",
+        "description": "A transform plugin that filters records using a custom Javascript provided in the plugin's config.",
+        "name": "ScriptFilter",
+        "properties": {
+            "script": {
+                "description": "Javascript that must implement a function 'shouldFilter' that takes a JSON object representation of the input record, and returns true if the input record should be filtered and false if not. For example: 'function shouldFilter(input) { return input.count > 100; }' will filter out any records whose 'count' field is greater than 100.",
+                "name": "script",
+                "required": true,
+                "type": "string"
+            }
+        },
+        "type": "transform"
+      }
+    ]
+
+.. _http-restful-api-artifact-delete:
+
+Deleting an Artifact
+--------------------
+To delete an artifact, submit an HTTP DELETE request::
+
+  DELETE <base-url>/namespaces/<namespace>/artifacts/<artifact-name>/versions/<artifact-version>
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<namespace>``
+     - Namespace ID
+   * - ``<artifact-name>``
+     - Name of the artifact.
+   * - ``<artifact-version>``
+     - Version of the artifact.
+
+Deleting an artifact is an advanced feature. If there are programs that use the artifact, those
+programs will not be able to start unless the artifact is added again, or the program application
+is updated to use a different artifact. 
+
+.. _http-restful-api-artifact-system-load:
+
+Load System Artifacts
+---------------------
+To load all system artifacts on the CDAP Master node(s), submit an HTTP POST request::
+
+  POST <base-url>/namespaces/system/artifacts
+
+This call will make the CDAP master scan the artifacts directly and add any new artifacts
+that it finds. Any snapshot artifacts will be re-loaded.
```

Comment:
this is the same for standalone and distributed.  Don't think we need to mention it if there is no difference.